### PR TITLE
*/releases, TOC: publish 3.0 GA release notes

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 

This PR updated the release notes for GA 3.0

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1540

### Which version does your change affect? 
V3.0/Dev

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/releases/3.0.0-ga.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+- Stability. TiDB 3.0 has demonstrated long-term stability for large-scale clusters with up to 150+ nodes and 300+TB of storage.
+- Usability. TiDB 3.0 has multi-facet improvements in usability, including standardized slow query logs, well-developed log file specification, and new features such as `EXPLAIN ANALYZE` and SQL Trace to save operation costs for users.
+- Performance. TiDB 3.0 has approximately 3.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
+- New features including Window Functions, Views (**Experimental**), partitioned tables, the plugin framework, pessimistic locking (**Experimental**), and `SQL Plan Management`.
+
+## TiDB
+
++ New Features
+    - Support Window Functions; compatible with all window functions in MySQL 8.0,     including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`,      `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```
Comment (ericsyh) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
    - Support Window Functions; compatible with all window functions in MySQL 8.0, including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`,      `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```

File: v3.0/releases/3.0.0-ga.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
++ Permission Management
+    - Perform permission check on the `ANALYZE`, `USE`, `SET GLOBAL`, and  `SHOW PROCESSLIST` statements
+    - Support Role Based Access Control (RBAC) (**Experimental**)
++ Server
+    - Optimize slow query logs:
+        - Restructure the log format
+        - Optimize the log content
+        - Optimize the log query method to support using the `INFORMATION_SCHEMA.SLOW_QUERY` and `ADMIN SHOW SLOW` statements of the memory table to query slow query logs
+    - Develop a unified log format specification with restructured log system to facilitate collection and analysis by tools
+    - Support using SQL statements to manage Binlog services, including querying status, enabling Binlog, maintaining and sending Binlog strategies.
```
Comment (ericsyh) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
    - Support using SQL statements to manage TiDB Binlog services, including querying status, enabling TiDB Binlog, maintaining and sending TiDB Binlog strategies.
```

File: dev/releases/3.0.0-ga.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+TiDB Ansible version: 3.0.0
+
+## Overview
+
+On June 28, 2019, TiDB 3.0.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
+
+- Stability. TiDB 3.0 has demonstrated long-term stability for large-scale clusters with up to 150+ nodes and 300+TB of storage.
+- Usability. TiDB 3.0 has multi-facet improvements in usability, including standardized slow query logs, well-developed log file specification, and new features such as `EXPLAIN ANALYZE` and SQL Trace to save operation costs for users.
+- Performance. TiDB 3.0 has approximately 3.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
```
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
- Performance. TiDB 3.0 has approximately 4.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
```

File: v3.0/releases/3.0.0-ga.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+TiDB Ansible version: 3.0.0
+
+## Overview
+
+On June 28, 2019, TiDB 3.0.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
+
+- Stability. TiDB 3.0 has demonstrated long-term stability for large-scale clusters with up to 150+ nodes and 300+TB of storage.
+- Usability. TiDB 3.0 has multi-facet improvements in usability, including standardized slow query logs, well-developed log file specification, and new features such as `EXPLAIN ANALYZE` and SQL Trace to save operation costs for users.
+- Performance. TiDB 3.0 has approximately 3.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
```
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
- Performance. TiDB 3.0 has approximately 4.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
```

**Comment by dcalvin** [comment]
PTAL @lilin90

**Comment by dcalvin** [comment]
PTAL @lilin90

File: v3.0/releases/3.0.0-ga.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+- Stability. TiDB 3.0 has demonstrated long-term stability for large-scale clusters with up to 150+ nodes and 300+ TB of storage.
+- Usability. TiDB 3.0 has multi-facet improvements in usability, including standardized slow query logs, well-developed log file specification, and new features such as `EXPLAIN ANALYZE` and SQL Trace to save operation costs for users.
+- Performance. TiDB 3.0 has approximately 4.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
+- New features including Window Functions, Views (**Experimental**), partitioned tables, the plugin framework, pessimistic locking (**Experimental**), and `SQL Plan Management`.
+
+## TiDB
+
++ New Features
+    - Support Window Functions; compatible with all window functions in MySQL 8.0, including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`,      `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```
Comment (ericsyh) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
    - Support Window Functions; compatible with all window functions in MySQL 8.0, including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`, `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```

File: dev/releases/3.0.0-ga.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+- Stability. TiDB 3.0 has demonstrated long-term stability for large-scale clusters with up to 150+ nodes and 300+ TB of storage.
+- Usability. TiDB 3.0 has multi-facet improvements in usability, including standardized slow query logs, well-developed log file specification, and new features such as `EXPLAIN ANALYZE` and SQL Trace to save operation costs for users.
+- Performance. TiDB 3.0 has approximately 4.5 times of performance improvements with TPC-C and 1.5 times with Sysbench. Thanks to the implementation of Views, TPC-H 50G Q5 and Q15 have both yielded normal results.
+- New features including Window Functions, Views (**Experimental**), partitioned tables, the plugin framework, pessimistic locking (**Experimental**), and `SQL Plan Management`.
+
+## TiDB
+
++ New Features
+    - Support Window Functions; compatible with all window functions in MySQL 8.0, including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`,      `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```
Comment (ericsyh) [inline dev/releases/3.0.0-ga.md]:
```suggestion
    - Support Window Functions; compatible with all window functions in MySQL 8.0, including `NTILE`, `LEAD`, `LAG`, `PERCENT_RANK`, `NTH_VALUE`, `CUME_DIST`, `FIRST_VALUE` , `LAST_VALUE`, `RANK`, `DENSE_RANK`, and `ROW_NUMBER`
```

File: dev/releases/3.0.0-ga.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
++ SQL Optimizer
+    - Optimize the `NOT EXISTS` subquery and convert it to `Anti Semi Join` to improve performance
+    - Optimize the constant propagation on the `Outer Join`, and add the optimization rule of `Outer Join` elimination to reduce non-effective computations and improve performance
+    - Optimize the `IN` subquery to execute `Inner Join` after aggregation to improve performance
+    - Optimize `Index Join` to adapt to more scenarios
+    - Improve the Partition Pruning optimization rule of Range Partition
+    - Optimize the query logic for `_tidb_rowid`to avoid full table scan and improve performance
+    - Match more prefix columns of the indexes when extracting access conditions of composite indexes if there are relevant columns in the filter to improve performance
+    - Improve the accuracy of cost estimates by using order correlation between columns
+    - Optimize `Join Reorder` based on the Greedy algorithm and the dynamic planning algorithm to improve accuracy for index selection using `Join`
```
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
    - Optimize `Join Reorder` based on the Greedy algorithm and the dynamic planning algorithm to improve the join order selection when joining multiple tables
```

File: v3.0/releases/3.0.0-ga.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
++ SQL Optimizer
+    - Optimize the `NOT EXISTS` subquery and convert it to `Anti Semi Join` to improve performance
+    - Optimize the constant propagation on the `Outer Join`, and add the optimization rule of `Outer Join` elimination to reduce non-effective computations and improve performance
+    - Optimize the `IN` subquery to execute `Inner Join` after aggregation to improve performance
+    - Optimize `Index Join` to adapt to more scenarios
+    - Improve the Partition Pruning optimization rule of Range Partition
+    - Optimize the query logic for `_tidb_rowid`to avoid full table scan and improve performance
+    - Match more prefix columns of the indexes when extracting access conditions of composite indexes if there are relevant columns in the filter to improve performance
+    - Improve the accuracy of cost estimates by using order correlation between columns
+    - Optimize `Join Reorder` based on the Greedy algorithm and the dynamic planning algorithm to improve accuracy for index selection using `Join`
```
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
    - Optimize `Join Reorder` based on the Greedy algorithm and the dynamic planning algorithm to improve the join order selection when joining multiple tables
```

File: dev/releases/3.0.0-ga.md (hunk 7 above)
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
    - Optimize `Join Order` based on the Greedy algorithm and the dynamic planning algorithm to speed up the join operation of multiple tables
```

File: v3.0/releases/3.0.0-ga.md (hunk 8 above)
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
    - Optimize `Join Order` based on the Greedy algorithm and the dynamic planning algorithm to speed up the join operation of multiple tables
```

File: dev/TOC.md [hunk 9]
```diff
@@ -325,6 +325,7 @@
 - [Roadmap](roadmap.md)
 + [Releases](releases/rn.md)
   + v3.0
+    - [3.0.0 GA](releases/3.0.0-ga.md)
```
Comment (dcalvin) [inline dev/TOC.md]:
```suggestion
    - [3.0 GA](releases/3.0-ga.md)
```

File: dev/releases/3.0.0-ga.md [hunk 10]
```diff
@@ -0,0 +1,218 @@
+---
+title: TiDB 3.0.0 GA Release Notes
```
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
title: TiDB 3.0 GA Release Notes
```

File: dev/releases/3.0.0-ga.md [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+category: Releases
+---
+
+# TiDB 3.0.0 GA Release Notes
```
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
# TiDB 3.0 GA Release Notes
```

File: dev/releases/3.0.0-ga.md [hunk 12]
```diff
@@ -0,0 +1,218 @@
...
+
+Release date: June 28, 2019
+
+TiDB version: 3.0.0
+
+TiDB Ansible version: 3.0.0
+
+## Overview
+
+On June 28, 2019, TiDB 3.0.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
```
Comment (dcalvin) [inline dev/releases/3.0.0-ga.md]:
```suggestion
On June 28, 2019, TiDB 3.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
```

File: v3.0/releases/3.0.0-ga.md [hunk 13]
```diff
@@ -0,0 +1,218 @@
+---
+title: TiDB 3.0.0 GA Release Notes
```
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
title: TiDB 3.0 GA Release Notes
```

File: v3.0/releases/3.0.0-ga.md [hunk 14]
```diff
@@ -0,0 +1,218 @@
...
+category: Releases
+---
+
+# TiDB 3.0.0 GA Release Notes
```
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
# TiDB 3.0 GA Release Notes
```

File: v3.0/releases/3.0.0-ga.md [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+
+Release date: June 28, 2019
+
+TiDB version: 3.0.0
+
+TiDB Ansible version: 3.0.0
+
+## Overview
+
+On June 28, 2019, TiDB 3.0.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
```
Comment (dcalvin) [inline v3.0/releases/3.0.0-ga.md]:
```suggestion
On June 28, 2019, TiDB 3.0 GA is released. The corresponding TiDB Ansible version is 3.0.0. Compared with TiDB 2.1, this release has greatly improved in the following aspects:
```

File: dev/releases/rn.md [hunk 16]
```diff
@@ -6,7 +6,7 @@ category: release
 # TiDB Release Notes
 
 ## 3.0
-
+- [3.0.0 GA](/releases/3.0.0-ga.md)
```
Comment (dcalvin) [inline dev/releases/rn.md]:
```suggestion
- [3.0 GA](/releases/3.0-ga.md)
```

File: v3.0/TOC.md [hunk 17]
```diff
@@ -325,6 +325,7 @@
 - [Roadmap](roadmap.md)
 + [Releases](releases/rn.md)
   + v3.0
+    - [3.0.0 GA](releases/3.0.0-ga.md)
```
Comment (dcalvin) [inline v3.0/TOC.md]:
```suggestion
    - [3.0 GA](releases/3.0-ga.md)
```

File: v3.0/releases/rn.md [hunk 18]
```diff
@@ -7,7 +7,7 @@ aliases: ['/docs/releases/rn/']
 # TiDB Release Notes
 
 ## 3.0
-
+- [3.0.0 GA](/releases/3.0.0-ga.md)
```
Comment (dcalvin) [inline v3.0/releases/rn.md]:
```suggestion
- [3.0 GA](/releases/3.0-ga.md)
```

**Pull request merged**